src/views/home/home.spec.jsx

```diff
@@ -43,7 +43,7 @@ describe('Home View component', () => {
     expect(home.find(EventsList).length).toEqual(1);
   });
 
-  it('should have a BlogPostsList component', () => {
-    expect(home.find(BlogPostsList).length).toEqual(1);
+  it('should not mount a BlogPostsList component', () => {
+    expect(home.find(BlogPostsList).length).toEqual(0);
```

Comment:
I think it would be great if we could add an event simulation to ensure that when something does happen, we should get a `<BlogPostList>` thought.  👍

Reply:
note: we should add it a new `it` block / test

Reply:
@thescientist13 : Sure. Will push now

Reply:
@thescientist13 : I just discovered that we can use shallow to test this Ref: https://gist.github.com/fokusferit/e4558d384e4e9cab95d04e5f35d4f913